src/components/sbb-link-list/sbb-link-list.tsx

```diff
@@ -8,20 +7,39 @@ import { InterfaceTitleAttributes } from '../sbb-title/sbb-title.custom.d';
  * list items with the links inside
  */
 
+let nextId = 0;
+
 @Component({
   shadow: true,
   styleUrl: 'sbb-link-list.scss',
   tag: 'sbb-link-list',
 })
 export class SbbLinkList {
-  private _guid: string;
+  private _hideTitleAndButton = false;
+
+  /** Sbb-Link elements */
+  private _links: HTMLSbbLinkElement[];
```

Comment:
Should this not be a `@State()` property to trigger a rerender on dynamically adding an sbb-link?